### DISCUSSION TITLE
hy on micropython

### DISCUSSION THREAD

**Discussion by tigadad** [hylang/hy#2378] [Ideas]
Does anyone else feel like getting Hy on Micropython; specifically the RP2040 (Pico).

**Comment by Kodiologist** [comment]
I'm not familiar with this project, but using Hy to generate Python code and running that sounds like it would be more feasible than running Hy on the hardware.

**Comment by tigadad** [comment]
I can't argue with that. The problem is that Micropython is not Python, but a subset. This also denies the opportunity for in-situ rule development.
